mantidproject/mantidimaging
# Update documentation within the user guide for the spectrum viewer

### Issue

Closes #2048 

### Description

Update Spectrum Viewer Documentation

### Testing 

* Proof Read to check for spelling errors
* Build Docs to check updated documentation is correctly formatted.
* Check URLs are correct.

### Acceptance Criteria 

- [ ] Documentation describes the spectrum viewer today.
- [ ] Documentation provides a good description of the spectrum viewer to someone who has never used the spectrum viewer who may have no prior experience working with tomography data. 
- [ ] No spelling or grammar errors
- [ ] All URLs work


### Documentation

`docs/release_notes/next/feature-2048-spectrum_documentation`
